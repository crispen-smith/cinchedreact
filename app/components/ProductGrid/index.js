/**
 *
 * ProductGrid
 *
 */

import React from 'react';
import propTypes from 'prop-types';
import styled from 'styled-components';

const PG = styled.div`
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: space-between;
  align-items: flex-start;
  align-content: flex-start;
  margin: 1rem;
  z-index: 100;
`;

const ProductGrid = props => <PG>{props.children}</PG>;

ProductGrid.propTypes = {
  children: propTypes.oneOfType([
    propTypes.arrayOf(propTypes.element),
    propTypes.string,
    propTypes.element,
  ]),
};

export default ProductGrid;
