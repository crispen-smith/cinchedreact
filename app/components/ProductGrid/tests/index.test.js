import React from 'react';
import renderer from 'react-test-renderer';

import ProductGrid from '../index';

describe('<ProcuctGrid />', () => {
  it('Renders a ProductGrid', () => {
    const tree = renderer.create(<ProductGrid>TEST</ProductGrid>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
