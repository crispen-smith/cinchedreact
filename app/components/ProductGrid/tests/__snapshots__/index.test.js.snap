// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProcuctGrid /> Renders a ProductGrid 1`] = `
<div
  className="ProductGrid-dKkFgu fwcJUv"
>
  TEST
</div>
`;
